src/pages/verify/e-ids/swedish-bankid.mdx

```diff
@@ -357,7 +357,7 @@ Criipto acts as a reseller for Swedbank, which means that you will have to go th
 
 import template from './files/swedbank-verksamhetsinfo-bankid-template.pdf';
 
-To order the actual certificate simply <a href={template} download>download this form</a> and return a filled out copy to <a href="mailto:support@criipto.com?subject=Swedish BankID for ...">support@criipto.com</a>
+To order the actual certificate simply <a href={template} download>download this form</a> and return a filled out copy to <a href="mailto:orders@criipto.com?subject=Swedish BankID for ...">orders@criipto.com</a>
```

Comment:
Unrelated to this PR, but since you're at it:
```suggestion
To order the actual certificate simply <a href={template} download>download this form</a> and return a filled out copy to <a href="mailto:orders@criipto.com?subject=Swedish BankID for ...">orders@criipto.com</a>.
```